rundoozer/rundoozer

```diff
@@ -214,10 +217,21 @@ def main():
 
     doozer_args = clean_args(doozer_args)
 
-    cfg = dotconfig.Config('rundoozer', 'settings',
-                           template=CLI_CONFIG_TEMPLATE,
-                           envvars=CLI_ENV_VARS,
-                           cli_args=args)
+    settings_file = None
+    if 'DOOZER_SETTINGS' in os.environ:
+        settings_file = os.environ['DOOZER_SETTINGS']
```

Comment:
Couldn't you do like....

`settings_file = os.environ.get('DOOZER_SETTINGS', None)`
?

Reply:
Yep... old habits. I know. I know.